velox/functions/prestosql/StringFunctions.h

```diff
@@ -507,4 +507,76 @@ struct StrLPosFunction : public StrPosFunctionBase<T, true> {};
 template <typename T>
 struct StrRPosFunction : public StrPosFunctionBase<T, false> {};
 
+template <typename T>
+struct FromBigEndian32 {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE
+  void callNullFree(
+      out_type<int32_t>& result,
+      const null_free_arg_type<Varbinary>& input) {
+    static const auto kTypeLength = sizeof(int32_t);
```

Comment:
Should this be constexpr instead ? Same comment below.

Reply:
Updated.